testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/naming/remote/ejb/RemoteNamingEjbTestCase.java

```diff
@@ -55,13 +59,15 @@
 import org.junit.Assert;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.jboss.as.test.integration.jca.JcaMgmtBase;
+
 
 /**
  * @author John Bailey, Ondrej Chaloupka
  */
 @RunWith(Arquillian.class)
 @RunAsClient
-public class RemoteNamingEjbTestCase {
+public class RemoteNamingEjbTestCase extends JcaMgmtBase{
```

Comment:
This test shouldn't subclass JcaMgmtBase just to pick up some utility code. In a later comment I'll explain how to avoid that.